.github/workflows/scanner-build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         # If this is updated, be sure to update architectures in push-manifests below.
-        goarch: ['amd64', 'arm64', 'ppc64le', 's390x']
```

Comment:
Just for testing purpose, maybe we can remove it if the team agree on this

Reply:
Will rebase with master when https://github.com/stackrox/stackrox/pull/9389/files is merged

Reply:
rebase